junit5/pom.xml

```diff
@@ -43,6 +43,12 @@
             <artifactId>assertj-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
It is just a test dependency (so no change to public API dependency tree) to avoid SFL4J warning that no impl is available.
We are already using it in other module https://github.com/xtf-cz/xtf/blob/master/core/pom.xml#L76-L80